src/Items/ItemChest.h

```diff
@@ -27,6 +27,8 @@ class cItemChestHandler:
 	}
 
 
+	/** We need an OnPlayerPlace override because we're processing neighbor chests and changing their metas,
```

Comment:
It uses `SetBlock` which deletes and re-creates the block entity, so it would destroy the chest contents.